### PR TITLE
Add the OSPO Alliance to our projects

### PR DESCRIPTION
**Pull request by bvhme**
Following today's announcement, text taken from the Eclipse Foundation press release.

### CODE REVIEW

**Comment by Ainali** [comment]
Should we expand the acronym "OSPO"? Or do we expect everyone that finds this page to know what it stands for?

**Review by Ainali — APPROVED** [review]
Looks good!

File: index.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ The game is also useful as a tool for visualizing how a current governance model
 A podcast where we interview people with interesting perspectives on public code.
 Recorded as a live stream and then edited to an audio podcast version.
 
+### [OSPO Alliance and OSPO.Zone](https://ospo.zone/)
+
+The OSPO Alliance aims to help companies and public institutions discover and understand open source, start benefiting from it across their activities and grow to host an Open Source Program Office (OSPO). It will be focused on the role of OSPOs as an aspect of organizational governance essential in non-technology sectors including education and public administration.
+
+This is an initiative we've taken with the OW2, Eclipse Foundation and OpenForum Europe.
```
Comment (ElenaFdR) [inline index.md:40]:
```suggestion
This is an initiative we've taken with OW2, Eclipse Foundation and OpenForum Europe.
```

**Review by ElenaFdR — COMMENTED** [review]
nice addition!

**Review by ElenaFdR — APPROVED** [review]
ready to go!

**Pull request merged**